src/Files.App/App.xaml

```diff
@@ -45,6 +45,9 @@
 							<SolidColorBrush x:Key="TabViewItemHeaderBackgroundSelected" Color="{StaticResource LayerOnMicaBaseAltFillColorDefault}" />
 							<SolidColorBrush x:Key="TabViewItemHeaderBackgroundPressed" Color="{StaticResource SubtleFillColorTertiary}" />
 							<SolidColorBrush x:Key="TabViewItemHeaderBackgroundPointerOver" Color="{StaticResource SubtleFillColorSecondary}" />
+
+							<Color x:Key="AppThemeFillColorAttention">#0070CB</Color>
+							<SolidColorBrush x:Key="AppThemeFillColorAttentionBrush" Color="{StaticResource AppThemeFillColorAttention}" />
```

Comment:
```suggestion
							<SolidColorBrush x:Key="App.Theme.FillColorAttentionBrush" Color="{StaticResource AppThemeFillColorAttention}" />
```
Using `.` makes it easier to select the correct brush from the suggestions drop down.